plugins/exporter.koplugin/target/html.lua

```diff
@@ -0,0 +1,62 @@
+local logger = require("logger")
+local slt2 = require("template/slt2")
+
+-- html exporter
+local HtmlExporter = require("base"):new {
+    name = "html",
+}
+
+local function format(booknotes)
```

Comment:
@uroybd: not familiar enough with this function. Please check if some stuff is old now that "normalization" happens inside json.

Reply:
This function creates a data structure that allowed me to show the chapter name display properly in the exported HTML.

The emitted structure for a book's clippings is such:

```lua
{
    -- Other fields except entries (e.g. title)
    chapters = {
        {
            title = "Chapter title",
            entries = {
                --all entries in that chapter
           }
       }
    }
      -- other chapters
}
```